scripts/apps/workspace/helpers/getLabelForFieldId.ts

```diff
@@ -4,9 +4,10 @@ import ng from 'core/services/ng';
 export const getLabelForFieldId = (fieldId, vocabularies) => {
     const labelMap = GET_LABEL_MAP();
     const field = vocabularies.find((obj) => obj._id === fieldId);
+    const categories = vocabularies.find((v) => v._id === 'categories');
 
-    if (fieldId === 'anpa_category') {
-        return vocabularies.find((v) => v._id === 'categories').display_name;
+    if (fieldId === 'anpa_category' && categories) {
```

Comment:
Only use boolean expressions inside `if` statements. `categories` is a vocabulary object, not a boolean, so you should use `categories != null` inside `if` stamenent - which will evaluate to a boolean.

Also `categories` is not the best naming choice since the variable represents a single vocabulary while variable name is in plural. A better name would be `categoriesVocabulary`.